waspc/e2e-test/Tests/WaspBuildTest.hs

```diff
@@ -0,0 +1,23 @@
+module Tests.WaspBuildTest (waspBuild) where
+
+import GoldenTest (GoldenTest, makeGoldenTest)
+import ShellCommands
+  ( cdIntoCurrentProject,
+    reformatPackageJson,
+    setDbToPSQL,
+    waspCliBuild,
+    waspCliNew,
+  )
+
+waspBuild :: GoldenTest
+waspBuild = do
+  let commands =
+        sequence
+          [ waspCliNew,
+            cdIntoCurrentProject,
+            setDbToPSQL,
+            waspCliBuild,
+            reformatPackageJson
+          ]
+
+  makeGoldenTest "waspBuild" commands
```

Comment:
Just curious, although it makes no big difference: why not merge it like this?

```hs
  makeGoldenTest "waspBuild" $ sequence
          [ waspCliNew,
            cdIntoCurrentProject,
            setDbToPSQL,
            waspCliBuild,
            reformatPackageJson
          ]
```

Reply:
Yeah that works, can update! 👍🏻